elb_log_ingestor/__version__.py

```diff
@@ -0,0 +1 @@
+VERSION = "0.0.1"
```

Comment:
If this is the first public release, it should be `1.0.0`.

Reply:
https://0ver.org/

Reply:
From the [about page](https://0ver.org/about.html):

> Version 0.0.1 of ZeroVer was published by Mahmoud Hashemi, with help from Moshe, Mark, Kurt, and other patient collaborators, on 2018-04-01. ZeroVer is satire, please do not use it. We sincerely hope no project release schedules were harmed as a result of this humble attempt at programmer humor.

I'm all for comedy, but not in versioning.

Reply:
in seriousness - I don't think a 1.0.0 is necessary to indicate a first release or a production ready release.

Reply:
If there's no intent to publish this on PyPI, then I'm fine with it. :)